src/components/tile/tile.component.js

```diff
@@ -24,33 +19,35 @@ const Tile = (props) => {
       <TileContent
         key={ key }
         width={ contentWidth }
+        isHorizontal={ isHorizontal(orientation) }
+        isVertical={ isVertical(orientation) }
+        pt={ isVertical(orientation) && (props.pt, props.py, p) }
+        pb={ isVertical(orientation) && (props.pb, props.py, p) }
+        pr={ isHorizontal(orientation) && (props.pr, props.px, p) }
+        pl={ isHorizontal(orientation) && (props.pl, props.px, p) }
       >
         {React.cloneElement(child, childProps)}
       </TileContent>
     );
   });
 
+
   return (
     <StyledTile
-      padding={ padding }
       tileTheme={ as }
-      orientation={ orientation }
       width={ width }
-      { ...rest }
-      { ...tagComponent('tile', props) }
+      data-component='tile'
+      isHorizontal={ isHorizontal(orientation) }
+      p={ p }
+      { ...props }
     >
       {wrappedChildren}
     </StyledTile>
   );
 };
 
-Tile.defaultProps = {
-  as: 'tile',
-  orientation: 'horizontal',
-  padding: 'M'
-};
-
 Tile.propTypes = {
+  ...propTypes.space,
```

Comment:
Same as above, could you add this comment please: `/** Styled system spacing props */`